.cci.jenkinsfile

```diff
@@ -37,18 +37,8 @@ codestyle: {
         ci/codestyle.sh
       """)
   }
-},
-unit: {
-  // this branch runs unit tests (both Rust-based and C-based) and makes sure we compile
-  // fine against the MSRV
-  def n = 5
-  cosaPod(buildroot: true, runAsUser: 0, memory: "2Gi", cpu: "${n}") {
-      checkout scm
-      shwrap("""
-        MAKE_JOBS=${n} CARGO_BUILD_JOBS=${n} ci/msrv.sh
```

Comment:
In addition to checking the MSRV, this would also run the Rust unit tests.

Reply:
I think it's OK to just do a full build here now if that's what's required?

Reply:
OK I started on that in https://github.com/coreos/rpm-ostree/pull/2491 but I have a bigger fix I'm trying too.